Makefile

```diff
@@ -14,6 +14,8 @@ DOCKER_BUF := docker run -v $(shell pwd):/workspace --workdir /workspace bufbuil
 
 export GO111MODULE = on
 
+GORELEASER_CONFIG      = .goreleaser.yaml
```

Comment:
Why does Makefile need to know the goreleaser config? Does this get used by anything in here?

Reply:
No, I thought it would be necessary
But after a few tests, I got to know that it is not necessary
Thanks @clevinson